admin/tabs/logging/viewer/LogViewerDisplay.js

```diff
@@ -47,6 +55,17 @@ class LogViewerDisplay extends Component {
         });
     }
 
+    timedRefreshLines = () => {
```

Comment:
Total nit -- use method here.  Reserving these fat arrow properties for event handlers in render methods.